Common/TableProducer/collisionConverter.cxx

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/AnalysisDataModel.h"
+
+using namespace o2;
+using namespace o2::framework;
+
+// Swaps covariance matrix elements if the data is known to be bogus (collision_000 is bogus)
+struct collisionConverter {
+  Produces<aod::Collisions_001> Collisions_001;
+
+  void process(aod::Collisions_000::iterator const& collision)
```

Comment:
Probably one could subscribe to the entire table and do the loop in process. Less function calls...